### ISSUE TITLE
No hearts when breeding animals (Railcraft bug)

### ISSUE THREAD

**Issue by mushroomhostage** [mushroomhostage/exphc#43]
When you right-click an animal with wheat to cause them to enter breeding mode, they do, but no hearts are displayed above the animal.

**Comment by mushroomhostage** [comment]
Does not occur with MCPC 91 without any mods or plugins. The hearts disappear as soon as I install Railcraft. Tested with Railcraft-5.2.4-MCPC-1.2.5-R2.zip.

**Comment by mushroomhostage** [comment]
Not a porting issue. Also reproduces with vanilla Railcraft_Server_5.2.4.zip (and does not repro without it).

**Comment by mushroomhostage** [comment]
In summary, reproduces with 5.2.4 vanilla, 5.2.4 r2 port, even 5.2.0 port (on ExpHC 1.2.5 rev2). Issue was reported on 2012/05/07 here: http://www.minecraftforum.net/topic/701990-125-railcraft-524-forge-smp/page__st__2920 - related to feed station?

**Comment by mushroomhostage** [comment]
Reported at http://www.minecraftforum.net/topic/701990-125-railcraft-524-forge-smp/page__st__3120__gopid__15466693#entry15466693

**Comment by mushroomhostage** [comment]
There's now a 5.3.2 port, should retest but the issue probably still remains.,

**Comment by mushroomhostage** [comment]
Still unfixed. 

```
 <Silentspy> if you give wheat to any animal
 <Silentspy> it wont show the hearts
 <Silentspy> + milk bucket dont work
```

**Comment by mushroomhostage** [comment]
Dev says is he going to look into this soon.

**Comment by mushroomhostage** [comment]
Closing, has been reported upstream and author of Railcraft says he will look into it. Annoying but a fairly minor issue, no longer worth tracking here.

**Comment by mushroomhostage** [comment]
http://railcraft.wikispaces.com/Version+History+%28Info%29
Fixed in Railcraft 5.4.0 FIX: Fixed a bug with the Crowbar that was causing entity interaction to terminate on the client in SMP. (No hearts, no milking, etc...)

**Comment by mushroomhostage** [comment]
Verified this is [fixed](http://forums.technicpack.net/threads/animal-breeding-broken.18559/) in Railcraft 5.4.7 using http://dl.dropbox.com/u/91705584/railcraft-5.4.7-mcpc-test3.zip - on 1.2.5rev5 test server.
